dc/s2n-quic-dc/events/map.rs

```diff
@@ -307,3 +307,42 @@ struct PathSecretMapIdCacheAccessed<'a> {
     #[bool_counter("hit")]
     hit: bool,
 }
+
+#[event("path_secret_map:cleaner_cycled")]
+#[subject(endpoint)]
+/// Emitted when the cleaner task performed a single cycle
+///
+/// This can be used to track cache utilization
+struct PathSecretMapCleanerCycled {
+    /// The number of Path Secret ID entries left after the cleaning cycle
+    #[measure("entries")]
+    entries: usize,
+
+    /// The number of Path Secret ID entries that were retired in the cycle
+    #[measure("retired_entries")]
+    retired_entries: usize,
+
+    /// The utilization percentage of the available number of entries after the cycle
+    #[measure("utilization", Percent)]
+    utilization: f32,
+
+    /// The utilization percentage of the available number of entries before the cycle
+    #[measure("utilization.initial", Percent)]
+    initial_utilization: f32,
```

Comment:
Are we missing a utilization for the addresses (or the IDs)? I wonder if we should have a consistent prefix/suffix to make it clearer that you're not looking at an aggregate and need to inspect both metrics and not just one of them?

Reply:
Yeah this is from the IDs. But I can change it to emit for both with the prefixes.